css/styles.css

```diff
@@ -0,0 +1,641 @@
+}
+
+.hero > * {
+  margin-bottom: 20px;
+}
+
+.hero__title {
+  font-size: 1.5rem;
+  color: var(--color-carmine-pink);
+}
+
+.about__hero-heading {
+  font-size: 1rem;
+  font-weight: 900;
+  text-transform: uppercase;
+  line-height: 1.5;
+  display: inline-block;
+  background:
+    linear-gradient(
+      to left,
+      red,
+      rgba(236, 67, 96, 0.863),
+      rgba(255, 0, 0, 0.26),
+      rgb(105, 44, 3),
+      #ec5242,
+      red,
+      rgb(133, 88, 4),
+      red
+    );
+  -webkit-background-clip: text;
+  color: transparent;
+  background-size: contain;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.hero__heading {
+  font-size: 1.5rem;
+  font-weight: 900;
+  text-transform: uppercase;
+  line-height: 1.5;
+  display: inline-block;
+  background:
+    linear-gradient(
+      to left,
+      red,
+      rgba(236, 67, 96, 0.863),
+      rgba(255, 0, 0, 0.26),
+      rgb(105, 44, 3),
+      #ec5242,
+      red,
+      rgb(133, 88, 4),
+      red
+    );
+  -webkit-background-clip: text;
+  color: transparent;
+  background-size: contain;
+}
+
+.hero__body {
+  background-color: var(--color-lighter-gray);
+  border: 2px solid var(--color-light-gray);
+  padding: 15px;
+  line-height: 1.5;
+}
+
+.hero__date {
+  font-weight: 900;
+  color: var(--color-charleston-green);
+  font-size: 1.6rem;
+}
+
+.hero__venue {
+  font-size: 15px;
+}
+
+.program {
+  background-color: var(--color-charleston-green);
+  padding: 50px 25px;
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.program__title {
+  text-align: center;
+  color: var(--color-lighter-gray);
+  margin-bottom: 50px;
+  font-size: 1.5rem;
+  position: relative;
+}
+
+.program__title::after {
+  position: absolute;
+  content: "";
+  width: 100px;
+  height: 3px;
+  background-color: var(--color-royal-orange);
+  top: 150%;
+  left: 50%;
+  display: block;
+  transform: translate(-50%, -50%);
+}
+
+.program__card-image {
+  width: 90%;
+}
+
+.program__card {
+  margin-bottom: 20px;
+  display: grid;
+  grid-template-columns: 1fr 2fr 4fr;
+  justify-content: space-between;
+  align-items: center;
+  gap: 20px;
+  background-color: rgb(254, 244, 244, 0.2);
+  padding: 15px;
+  cursor: pointer;
+}
+
+.program__card:hover {
+  transform: scale(1.05);
+  transition: all 0.5s ease-in;
+}
+
+.program__card-title {
+  font-size: 1rem;
+  color: var(--color-royal-orange);
+}
+
+.program__card-body {
+  font-size: 13px;
+  justify-self: flex-start;
+  color: var(--color-lighter-gray);
+  line-height: 1.5;
+}
+
+.program__btn {
+  background-color: var(--color-royal-orange);
+  padding: 20px;
+  border: none;
+  margin-top: 20px;
+  font-family: inherit;
+  color: var(--color-lighter-gray);
+  font-size: 1.5rem;
+}
+
+.feature-speakers {
+  padding: 40px 20px;
+}
+
+.feature-speakers__container-title {
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.feature-speakers__card {
+  display: none;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 5px;
+  margin-bottom: 10px;
+}
```

Comment:
Kindly update your featured speakers section on both mobile and desktop to match the behance design better🙏

## Mobile

Your design(mobile)
<img width="293" alt="image" src="https://user-images.githubusercontent.com/37116322/232930444-3d7db135-9456-476f-a8c2-08e95442163c.png">

Behance design mobile
<img width="290" alt="image" src="https://user-images.githubusercontent.com/37116322/232930490-23a2dad7-39ed-4955-9489-941104e092b7.png">

A few things to note:
- Please include the orange underline under the title. This helps separate it from the rest of the content
- Please increase the spacing between each of the cards

## Desktop

Your design(destop)
<img width="959" alt="image" src="https://user-images.githubusercontent.com/37116322/232930920-6a7932d5-aee9-425c-ba62-1e6bc69b381a.png">

Behance design(desktop)
<img width="626" alt="image" src="https://user-images.githubusercontent.com/37116322/232931129-8f75ac3f-beb4-46f9-aaec-d0d38c371346.png">

A few things to note:
- Please increase the width of the cards to the paragraph text is not truncated